2022/src/day10.scala

```diff
@@ -0,0 +1,48 @@
+package day10
+
+import locations.Directory.currentDir
+import inputs.Input.loadFileSync
+
+import Direction.*
+
+def INPUT(): String = loadFileSync(s"$currentDir/../input/day10")
+
+enum Command {
+  case NOOP
+  case ADDX(X: Int)
+}
+
+export Command.*
+
+def commandsIterator: Iterator[Command] = for (line <- INPUT.linesIterator) yield line.strip match {
+  case "noop" => NOOP
+  case s"addx $x" if x.toIntOption.isDefined => ADDX(x.toInt)
+  case _ => throw IllegalArgumentException(s"Invalid command '$line''")
+}
+
+val REGISTER_START_VALUE = 1
```

Comment:
same here, please refactor to UpperCamelCase, e.g
```suggestion
val RegisterStartValue = 1
```